### PR TITLE
Spelling corrections.

### PR DESCRIPTION
**Pull request by Chaitanyassr**
Fix issue = #32 , #34 , #35 
Updated You as your in within practice in Facilitate Inclusively section.
It is present in the third point of "try".
- page link - https://guide.inclusivedesign.ca/practices/FacilitateInclusively.html

also updated specic to specific in tools in use case within try section.
- page link - https://guide.inclusivedesign.ca/tools/UseCases.html

also change "exerise" to "exercise" in "Activities" within cause and effect.
- page link - https://guide.inclusivedesign.ca/activities/CauseAndEffect.html

### CODE REVIEW

File: src/documents/practices/FacilitateInclusively.html.md.handlebars [hunk 1]
```diff
@@ -15,7 +15,7 @@ Thinking beyond technical limitations and avoiding getting bogged down in techni
 ## Try
 * Use accessible communication tools.
 * Consider how to support remote participation.
-* Make sure you space is physically accessible.
+* Make sure your space is physically accessible.
 * Ensure that everyone has the necessary background information.
```
Comment (BlueSlug) [inline src/documents/practices/FacilitateInclusively.html.md.handlebars:19]: Perhaps it makes more sense to change the tense of these verbs to present participle, so something like
```
Using...
Considering...
Making sure...
Ensuring...
```

Comment (Chaitanyassr) [reply src/documents/practices/FacilitateInclusively.html.md.handlebars:19]: Yes, changing these would mean that we have to change all the sentences which would be approximately 100lines of change. Should I proceed?

Comment (Chaitanyassr) [reply src/documents/practices/FacilitateInclusively.html.md.handlebars:19]: Okay! I would be creating a seperate PR for this.

Comment (Chaitanyassr) [reply src/documents/practices/FacilitateInclusively.html.md.handlebars:19]: new pr for this - https://github.com/inclusive-design/guide.inclusivedesign.ca/pull/36
thank you.

**Comment by jhung** [comment]
Fixed in PR #40 .

**Pull request closed without merging**